src/theory/quantifiers/sygus/sygus_eval_unfold.cpp

```diff
@@ -164,6 +166,12 @@ void SygusEvalUnfold::registerModelValue(Node a,
             do_unfold = true;
           }
         }
+        else if (options().quantifiers.sygusEvalUnfoldMode
+                 == options::SygusEvalUnfoldMode::SINGLE)
+        {
+          // do single step for all
+          do_unfold = true;
```

Comment:
this is changing behavior?

Reply:
This is a new mode, yes.  The default behavior has not changed.